translation/en/RedirectedResources/assets/abdata/communication/info_25/communication_off_33/translation.txt

```diff
@@ -1,551 +1,551 @@
-//「ンッ……待って、それ以上は、だめよ……」=
-//「見る必要があるのは私の心じゃない？　コイカツ部の部長ならお手の物でしょ？　ふふふ」=
-//「っ！…………」=
-//「意外と手を握るのって……大胆な行為よね」=
-//「恋をするって大変よね。昔はなんともなかったことが、急に大事件になっちゃうんだもの」=
-//「もう……するなら最後までして……」=
-//「ねえ……私から言わせるつもり？」=
-//「もう好きにして……」=
-//「くす。お触り１回１０００円ね……冗談よ、お財布はしまって」=
-//「兄がいたらこんな風に撫でてくれていたのかしらね」=
-//「なんだか撫で方がてなれてきたわね」=
-//「触り方、エロいわよ」=
-//「それで、あなたはどうしたいの？」=
-//「こうすると私がほしくなるのを知っててしてるんでしょ？　いじわるな人」=
-//「くすっ。そういうことは、喧嘩が終わってから、ね？」=
-//「あなたの前ではついついお喋りになっちゃうわね」=
-//「なんだか会話のキャッチボールというより、会話の前戯って感じね、ふふふ」=
-//「こうやって楽しく話していると、私たちって本当に付き合ってるんだって思って、嬉しくなるの。もっと話しましょ」=
-//「……ふふ……」=
-//「ふふふ」=
-//「今のはちょっと見過ごせないわね。お互いに分かり合うためだし、私怒っちゃうわ、ふふっ」=
-//「あら、そろそろ行かなきゃ。早く私の怒りを鎮めないと、いつまでもエッチできないわよ？　ふふふ」=
-//「幸せってこういう時間を言うのよね、きっと。またこうやってお喋りしましょ」=
-//「もう、ここでお別れなんて生殺しじゃない。次はこの火照りを鎮めてね？」=
-//「うふふ、またね」=
-//「あら、もう？　じゃあ、また都合のいいときに話しかけてね。待ってるわ」=
-//「ええ、また今度ね」=
-//「……そう。私の想いは伝わらなかったってことで、いいのね？　それなら仕方ないわね。別れましょうか、私たち」=
-//「ねえ、もう別れましょうか。私がいなくなっても、あなたは両手に花状態でしょ？　コイカツ部、頑張ってね」=
-//「……ふふ、恋人同士で溶け合うように気持ちよくなりましょ」=
-//「ねえ、カップルの営みをしない？　ふふふ」=
-//「今の私、とってもエッチだから、期待していてね？」=
-//「ふぅん。もしかして先約が入ってる？　それならタマタマの中身は溜めておかないとだものね？　ふふふ」=
-//「ここで[H名]さんとHしたい」=
-//「もう、スケベなんだから。じゃあ、学園新聞の一面記事に変態カップルとして載っちゃうぐらい激しくしないとね」=
-//「みんなの前でエッチしたいの？　うーん、それならもっと私の身体を開発しないとね？」=
-//「ええ、いいわよ。もし見られても、コイカツの一環ですって言えばいいものね」=
-//「こういうところでしたいなら、私をもっと淫乱女に仕立てあげないと。それはあなたの仕事、じゃなくて部活の役割でしょ？」=
-//「ええ、お昼にしましょうか。恋人とのランチ、いい響きよね」=
-//「そろそろランチにしない？」=
-//「ふふ、じゃあ行きましょうか」=
-//「あら、そう。それじゃ、また今度誘うわ」=
-//「ええ、世界一楽しい部活を始めましょ」=
-//「忙しいとは思うけど、もし私に時間をくれるならコイカツしない？」=
-//「あら、ダメ元でも言ってみるものね。それじゃ、楽しいコイカツのはじまりよ」=
-//「やっぱり忙しいわよね。美容室みたいに予約できたらいいのに、ふふふ」=
-//「ええ、いいわよ。ちょっと寄り道してもいいかしら？」=
-//「ねえ、よかったら一緒に帰らない？　それとも忙しいかしら」=
-//「本当？　ふふ、じゃあ仕度をしてくるわ」=
-//「そう。大丈夫、それなら友だちと帰るから。コイカツ頑張ってね、応援してるわ」=
-//「ふふふ、この商品はキャンセルできないけど大丈夫？　なんてね」=
-//「ねえ、今度デートに行かない？　１００人待ちでも大丈夫よ」=
-//「いいのね？　じゃあ遠慮なくスケジュール帳に入れちゃうわよ」=
-//「予想はしてたわ。列に並んでるから、順番が来たら声をかけてね」=
-//「恋人同士でお勉強？　イチャイチャしながらでもいいならお付き合いするわ、ふふふ」=
-//「よかったら一緒にお勉強しない？」=
-//「ふふ、濃密な時間にしましょうね」=
-//「あらそう。じゃあまた時間のあるときにしましょ」=
-//「いいわよ。あなたのことだから、ちょっと下心があるんでしょ？　人のことは言えないけどね」=
-//「少し運動につきあってくれないかしら？」=
-//「ありがとう。体型維持のために身体を動かしたかったの」=
-//「そう。気分じゃないなら、仕方ないわね」=
-//「付いてきてもらってもいい？」=
-//「恋人同士で学園散歩？　素敵ね」=
-//「謝ってくれたらどこにでも付いていくわよ。暗がりでも遊園地でも」=
-//「ええ」「いいわよ」「あなたがそう言うなら」「いいんじゃない？」「名案ね」=
-//「だめよ」「どうかしら」「違うんじゃない？」「気分じゃないの」「そうねぇ」=
-//「ふふ、それで？」「へぇ」「ふぅん」「そうなの」「ふふふ」=
-//「あら」「え？」「そうだったの」「びっくりね」「驚いたわ」=
-//「うふふ♪」「嬉しいわ」「私には勿体無いわ」「いいの？」「幸せね」=
-//「もう」「怒るわよ」「挑戦的じゃない」「口には気を付けないとね」「どういうこと？」=
-//「ふぅん……」「しょうがない人ね」「哀しいわぁ」「釣れないのね」「ふぅ……」=
-//「くすくすっ♪」「楽しいわね」「面白いじゃない」「やだもう」「はいはい」=
-//「…………」「もう、ずるいわ」「照れるからやめて」「あなた、わざとでしょ」「人の気も知らないで」=
-//「ねえ、ちょっといい？」「そういえば……」「あのね」「そうそう」「ねえ」=
-//「ねぇ、[Hあだ名]」=
-//「ふふ。なあに、[Pあだ名]」=
-//「ふふ、相手が怒ってるときに愛称で呼ぶと煽ってるように聞こえるわよ」=
-//「ふふふ……[Pあだ名]」=
-//「どうしたの？　[Hあだ名]」=
-//「好きよ」=
-//「ふふ、今だけは独り占めしちゃってもいいのよね？　[Pあだ名]」=
-//「ねぇ、今から愛を育まない？ [Pあだ名]」=
-//「あら、時間があるなら話さない？」=
-//「あら、[Pあだ名]。時間があるなら話さない？」=
-//「もちろん。歓迎するよ！」=
-//「ふふ、嬉しいわ。よろしくお願いするわね、部長さん？」=
+「あら、ここ女子トイレよ？」=Oh my, this is the girl's toilet, you know?
+「女の子の生態でも調べにきたの？ くすくすっ」=Came to check out even girls' private territory? *Giggle*
```

Comment:
Looks like spaces got changed in jp text

Reply:
re-dump. Old dump for personality failed to load text ingame due to problems with spaces